app/assets/javascripts/discourse/models/category-list.js.es6

```diff
@@ -52,10 +52,13 @@ CategoryList.reopenClass({
           const unit = I18n.t(statPeriod);
           if (stat > 0) {
             c.stat = `<span class="value">${stat}</span> / <span class="unit">${unit}</span>`;
-            c.statTitle = I18n.t("categories.topic_stat_sentence", {
-              count: stat,
-              unit: unit
+
+            const stringToTranslate =
+              "categories.topic_stat_sentence" + "_" + statPeriod;
```

Comment:
You can simplify this even more by using template literals:

```es6
            c.statTitle = I18n.t(
              `categories.topic_stat_sentence_${statPeriod}`,
              {
                count: stat
              }
            );
```

Reply:
@mrfinch Can you change this to use a template literal? The rest of the PR looks fine.

Reply:
@gschlager  done.

Thanks